ansible-scylla-node/tasks/RedHat.yml

```diff
@@ -32,7 +32,38 @@
         name: kernel-ml
         state: present
         lock_timeout: 60
-  when: elrepo_kernel == True and install_type == 'online' and (ansible_distribution == "CentOS" or ansible_distribution == "RedHat")
+      register: kernel_installed
+
+    - name: Configure grub2 to boot first entry
```

Comment:
this is needed only for Centos 7, for Centos 8 it's not - will it cause a problem in Centos 8 ?